.github/ISSUE_TEMPLATE/example_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Example request
+about: Use this template to track new examples requests.
+title: ''
+labels: example, example request
+assignees: ''
+
+---
+
+You would like to see a new example being implemented by the Flax core team or the community? Please let us know by filling the following template.
+
+### Description of the model to be implemented
+
+
+### Dataset the model could be used on
+
```

Comment:
Let's also add a section asking for reference implementations in other frameworks